[CUBRIDMAN-215] modify the manual by tech div's requirement (#513)
CUBRID/cubrid-manual: en/admin/admin_utils.rst

flashback
---------

> The **cubrid flashback** utility is used to get SQL statements to rewind a specific committed transaction, and can only be executed by the **DBA** user. The system parameter **supplemental_log** must be turned on, and flashback is supported only for DML executed after the **supplemental_log** is set. The utility is performed in only client/server mode and can't be performed more than one at same time. ::

Reviewer: ```suggestion
The **cubrid flashback** utility is used to get SQL statements to rewind a specific committed transaction, and can only be executed by the **DBA** user. The system parameter **supplemental_log** must be turned on, and flashback is supported only for DML executed after the **supplemental_log** is set. This utility can only be performed in client/server mode and cannot be executed more than once simultaneously. ::
```

Author: thank you, i will apply your opinion.
